core/tauri-utils/src/config.rs

```diff
@@ -1269,7 +1269,7 @@ pub struct WindowAllowlistConfig {
   /// Allows maximizing the window.
   #[serde(default)]
   pub maximize: bool,
-  /// Allows unmaximizing the window.
+  /// Allows restoring the window.
```

Comment:
Notable (happy to drop this or anything else)

Reply:
unmaximizing is better here to match the property and the function name too